app/krs.js

```diff
@@ -97,6 +97,10 @@ exports.provisionKey = co(function *(req) {
     throw utils.ErrorResponse(400, 'coin type required');
   }
 
+  if (!process.config.supportedcoins.includes(req.body.coin)) {
+    throw utils.ErrorResponse(400, 'unsupported coin');
```

Comment:
Hindsight is 20/20 but this makes me realize that platform is gonna say "unsupported coin" for any 400 now, it probably should just forward the error message on